tools/pot/openvino/tools/pot/data_loaders/image_loader.py

```diff
@@ -54,9 +58,11 @@ def get_layout(self, input_node=None):
             self._layout = Layout(self._layout)
             return
 
-        if input_node:
-            layout_from_ir = input_node.graph.graph.get('layout', None)
+        if input_node and input_node.graph.meta_data.get('layout', None) not in [None, '()']:
+            layout_from_ir = get_layout_values(input_node.graph.meta_data.get('layout', None))
```

Comment:
Not sure, but a number of incoming models can be without the `meta_data` section. I'm talking about models that were serialized using runtime methods (GNA, for example). Maybe we just need to get some kind of confirmation that `meta_data` will exists always. cc @AlexKoff88

Reply:
@KodiaqQ, does your commnet relevant to the image loader? As far as I know, meta-information is saved during the serialization. But probably someone from the ngraph team should comment here: @ilyachur, @GlebKazantaev?

Reply:
I suggest accepting this a wait for the validation of Simplified mode on a wide range of models

Reply:
Per discussion with @achetver, it seems that this is a valid case and we should check the availability of meta info in the model before trying to access it. @IndiraSalyahova, can you please change the code accordingly?

Reply:
@AlexKoff88 this condition was added. `and hasattr(input_node.graph, 'meta_data') `